pkg/cluster/inventory.go

```diff
@@ -164,16 +190,15 @@ func (i *DefaultInventory) createCluster(contractVersion int64, cluster *keb.Clu
 	}
 
 	//create new version
-	q, err := db.NewQuery(i.Conn, newClusterEntity, i.Logger)
+	q, err := db.NewQueryGorm(i.Conn, newClusterEntity, i.Logger)
 	if err != nil {
 		return nil, err
 	}
-	err = q.Insert().Exec()
+	newDbEntity, err := q.Insert(inventoryClusters{})
```

Comment:
is it okay that we do an empty insert here? what is the context here?

Reply:
This is not an empty insert, please have a look at the implementation of the Insert function. It creates a new row in the given table using the default values.